Migration.md

```diff
@@ -645,6 +672,20 @@ ABI was broken for `sdf::Element`, and restored on version 11.2.1.
     `//world/joint` and `//state/insertions/joint` can represent inserted
     `//world/joint` elements.
 
+### Deprecations
+
+1. **joint_state.sdf**:
+    + `//joint_state/angle` is deprecated in favor of `//axis_state/position`
+      and  `//axis2_state/position`.
+
+1. **link_state.sdf**:
+    + `//link_state/velocity` is deprecated in favor of `//link_state/angular_velocity`
+      and  `//link_state/linear_velocity`.
+    + `//link_state/acceleration` is deprecated in favor of `//link_state/angular_acceleration`
+      and  `//link_state/linear_acceleration`.
+    + `//link_state/wrench` is deprecated in favor of `//link_state/angular_wrench`
+      and  `//link_state/linear_wrench`.
```

Comment:
I've commonly seen the term wrench refer to the 6D vector that represents torques and forces. If we split them into two 3D vectors, would it make sense to call them torque and force instead of angular_wrench and linear_wrench?

Reply:
https://github.com/gazebosim/sdformat/pull/1461/commits/8521907ece4d3b4772846f50b19207543f930280

Reply:
I updated the pull request description with `force` and `torque` as well